docs/source/architecture.md

```diff
@@ -0,0 +1,73 @@
+Architecture
+============
+
+![NVTabular Workflow](../images/nvt_workflow.png)
+
+The NVTabular engine uses the [RAPIDS](http://www.rapids.ai) [Dask-cuDF library](https://github.com/rapidsai/dask-cuda), which provides the bulk of the functionality for accelerating dataframe operations on the GPU and scaling across multiple GPUs. NVTabular provides functionality commonly found in deep learning recommendation workflows, allowing you to focus on what you want to do with your data, and not how you need to do it. NVTabular also provides a template for our core compute mechanism, which is referred to as Operations (ops), allowing you to build your own custom ops from cuDF and other libraries.
+
+Once NVTabular is installed, you can set up a workflow as follows:
+
+```python
+import nvtabular as nvt
+workflow = nvt.Workflow(
+    cat_names=["user_id", "item_id", "city"],
+    cont_names=["age", "time_of_day", "item_num_views"],
+    label_name=["label"]
+)
```

Comment:
This seems to refer to the previous (0.3) api - we should update to the latest api.

The 'HowItWorks.md' in the main repo has been updated with these changes already